rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/DatabaseSanitation/Sw360DatabaseSanitationService.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Siemens AG, 2023-2024.
+ * Part of the SW360 Portal Project.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.eclipse.sw360.rest.resourceserver.DatabaseSanitation;
+
+import static org.eclipse.sw360.datahandler.common.CommonUtils.allAreEmptyOrNull;
+import static org.eclipse.sw360.datahandler.common.CommonUtils.oneIsNull;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.thrift.TException;
+import org.apache.thrift.protocol.TCompactProtocol;
+import org.apache.thrift.protocol.TProtocol;
+import org.apache.thrift.transport.THttpClient;
+import org.apache.thrift.transport.TTransportException;
+import org.eclipse.sw360.datahandler.thrift.RequestStatus;
+import org.eclipse.sw360.datahandler.thrift.SW360Exception;
+import org.eclipse.sw360.datahandler.thrift.components.ComponentService;
+import org.eclipse.sw360.datahandler.thrift.licenses.LicenseService;
+import org.eclipse.sw360.datahandler.thrift.projects.ProjectService;
+import org.eclipse.sw360.datahandler.thrift.users.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.data.rest.webmvc.ResourceNotFoundException;
+import org.springframework.hateoas.EntityModel;
+import org.springframework.http.converter.HttpMessageNotReadableException;
+import org.springframework.stereotype.Service;
+
+import lombok.RequiredArgsConstructor;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class Sw360DatabaseSanitationService {
+
+    @Value("${sw360.thrift-server-url:http://localhost:8080}")
+    private String thriftServerUrl;
+    
+    public Map<String, Object> dublicateIdentifiers(User sw360User, HttpServletRequest request) throws TException {
```

Comment:
parameters `sw360User`  and `request` never  used. Kindly remove those.

Reply:
done.